lib/find-visualstudio.js

```diff
@@ -389,6 +389,10 @@ class VisualStudioFinder {
       ret.versionYear = 2022
       return ret
     }
+    if (ret.versionMajor === 18) {
+      ret.versionYear = 2025
```

Comment:
Is this correct?!?

Reply:
Is Visual Studio 2025 published yet? I don't think VS version is aligned with Windows version. AFAICT, VS 2025 should be around November of 2025 to December of 2025.

Reply:
You are correct.  Windows 2025 currently ships with Visual Studio 2022.

`vswhere -latest -property installationPath`
```
C:\Program Files\Microsoft Visual Studio\2022\Enterprise
```